Marlin/Marlin_main.cpp

```diff
@@ -5276,24 +5254,24 @@ inline void gcode_G28() {
             SERIAL_EOL;
           }
         }
-        if (test_precision != 0.0) {            // !forced end
-          if (zero_std_dev >= test_precision) {
+        if (test_precision != 0.0) {                                                             // !forced end
+          if (zero_std_dev >= test_precision) {                                                  // end iterations
             SERIAL_PROTOCOLPGM("Calibration OK");
-            SERIAL_PROTOCOLLNPGM("                                   rolling back 1");
-            LCD_MESSAGEPGM("Calibration OK");
+            SERIAL_PROTOCOLPGM("                                    rolling back.");
             SERIAL_EOL;
+            LCD_MESSAGEPGM("Calibration OK");
           }
-          else {                                // !end iterations
+          else {                                                                                 // !end iterations
```

Comment:
No need for extra spaces here. If your editor is causing this stuff, please change your editor settings.